src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +58,29 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
-
   return (
-    <>
-      <h4 className="admin-page-title">You are editing venue: {venue.name}</h4>
-      <BannerAdmin venueId={venueId} venue={venue} />
-      {isIframeVenue && <IframeAdmin venueId={venueId} venue={venue} />}
```

Comment:
Why was `IframeAdmin` removed from here? That doesn't seem right..?

Reply:
seems to be removed in the design as well https://www.figma.com/file/oX3Px4vYjjUkOZYSqcCgWI/Sparkle-Banner-Announcements?node-id=3%3A6

Reply:
Maybe sync with @sparkletown/product and find out the ‘why’ of that, if it was explicitly intended. I have a feeling it may be a case of the designs not reflecting the reality of the codebase rather than an explicit desire

Reply:
@0xdevalias as per discussion with Sofi this `banner` feature is to be replaced by announcement one, so that should explain the removal of this code piece

Reply:
IframeAdmin is entirely different to banneradmin..

Reply:
@alexey-petrov, please, get `<IframeAdmin />` back. It's about changing the URL for iframe venues.
![image](https://user-images.githubusercontent.com/63313732/133635982-a9d644dd-90c0-4227-844a-12403f853d43.png)